packages/adblocker/src/engine/engine.ts

```diff
@@ -1443,29 +1468,31 @@ export default class FilterEngine extends EventEmitter<EngineEventHandlers> {
     if (request.isSupported) {
       // Check the filters in the following order:
       // 1. $important (not subject to exceptions)
-      // 2. redirection ($redirect=resource)
+      // 2. redirection ($removeparam and $redirect=resource)
```

Comment:
Is this comment up-to-date?

Reply:
Fixed in 22cd3d71075ce5279916e6b88f56fbbde657beee